esridump/dumper.py

```diff
@@ -297,11 +316,33 @@ def _scrape_an_envelope(self, envelope, outSR, max_records):
             for feature in features:
                 yield feature
 
+    def _esri2geojson(self, feature):
```

Comment:
It's weird to have a wrapper function that is named the same as the function it's calling. Would it make sense to call this `reproject_and_convert()` or something instead?